camp/nfutsum/Twisty/src/main/java/frc/robot/subsystems/NaherNeoMotor.java

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+public class NaherNeoMotor extends SubsystemBase {
```

Comment:
missing definition comment bloc - applies to any public method, constructor and class